frontend/src/components/UserSearch.vue

```diff
@@ -92,6 +191,35 @@ export default {
           window.scrollTo(0, 0);
         })
         .finally(() => (this.userSearchLoadingStatus = false));
+    },
+    searchUserAdvanced: function() {
+      let params = '';
+      params = this.addQueryParameter(params, "lastName", this.lastNameInput)
+      params = this.addQueryParameter(params, "firstName", this.firstNameInput)
+      params = this.addQueryParameter(params, "username", this.usernameInput)
+      params = this.addQueryParameter(params, "email", this.emailInput)
+      params = this.addQueryParameter(params, "org", this.organizationInput)
+
+      UsersRepository.get(
+          "?briefRepresentation=true&first=0&max=300" + params
```

Comment:
Using briefRepresentation=true makes the search by Org. ID yield no results since the UserRepresentation returned by KC does not contain 'attributes'. Should this be overridden by the API when "org" parameter is present?

Reply:
never mind. I see that the final commit uses briefRepresentation=false

Reply:
No sorry, that's a mistake by me. I messed up my rebase and lost some of my changes, I thought I fixed it all but I missed this. I'll update to set briefRepresentation=false